x/slashing/simulation/decoder_test.go

```diff
@@ -36,7 +37,7 @@ func TestDecodeStore(t *testing.T) {
 		Pairs: []kv.Pair{
 			{Key: types.ValidatorSigningInfoKey(consAddr1), Value: cdc.MustMarshal(&info)},
 			{Key: types.ValidatorMissedBlockBitmapKey(consAddr1, 6), Value: missed},
-			{Key: types.AddrPubkeyRelationKey(delAddr1), Value: bz},
+			{Key: append(types.AddrPubkeyRelationKeyPrefix, address.MustLengthPrefix(delAddr1)...), Value: bz},
```

Comment:
is this still need to be tested?

Reply:
nope, forgot to remove this. Thanks for pointing out.